mteb/tasks/Retrieval/dan/DanFeverRetrieval.py

```diff
@@ -0,0 +1,221 @@
+      Derczynski, Leon",
+    editor = "Dobnik, Simon  and
+      {\O}vrelid, Lilja",
+    booktitle = "Proceedings of the 23rd Nordic Conference on Computational Linguistics (NoDaLiDa)",
+    month = may # " 31--2 " # jun,
+    year = "2021",
+    address = "Reykjavik, Iceland (Online)",
+    publisher = {Link{\"o}ping University Electronic Press, Sweden},
+    url = "https://aclanthology.org/2021.nodalida-main.47",
+    pages = "422--428",
+    abstract = "We present a dataset, DanFEVER, intended for multilingual misinformation research. The dataset is in Danish and has the same format as the well-known English FEVER dataset. It can be used for testing methods in multilingual settings, as well as for creating models in production for the Danish language.",
+}
+""",
+        n_samples={"train": 8897},
+        avg_character_length={
+            "train": {
+                "average_document_length": 312.1117274167987,
+                "average_query_length": 50.26957476855484,
+                "num_documents": 2524,
+                "num_queries": 6373,
+                "average_relevant_docs_per_query": 0.48721167425074535,
+            }
+        },
+        task_subtypes=["Claim verification"],
+    )
+
+    def load_data(self, **kwargs):
+        """Load dataset from HuggingFace hub"""
+        if self.data_loaded:
+            return
+        self.dataset = datasets.load_dataset(**self.metadata.dataset)  # type: ignore
+        self.dataset_transform()
+        self.data_loaded = True
+
+    def dataset_transform(self) -> None:
+        """And transform to a retrieval datset, which have the following attributes
+
+        self.corpus = Dict[doc_id, Dict[str, str]] #id => dict with document data like title and text
+        self.queries = Dict[query_id, str] #id => query
+        self.relevant_docs = Dict[query_id, Dict[[doc_id, score]]
+        """
+        self.corpus = {}
+        self.relevant_docs = {}
+        self.queries = {}
+        text2id = {}
+
+        for split in self.dataset:
+            self.corpus[split] = {}
+            self.relevant_docs[split] = {}
+            self.queries[split] = {}
+
+            ds = self.dataset[split]
+            claims = ds["claim"]
+            evidences = ds["evidence_extract"]
+            labels = ds["label"]
+            class_labels = ds.features["label"].names
+
+            for claim, evidence, label_id in zip(claims, evidences, labels):
+                claim_is_supported = class_labels[label_id] == "Supported"
+
+                sim = (
+                    1 if claim_is_supported else 0
+                )  # negative for refutes claims - is that what we want?
+
+                if claim not in text2id:
+                    text2id[claim] = str(len(text2id))
+                if evidence not in text2id:
+                    text2id[evidence] = len(text2id)
+
+                claim_id = "Q" + str(text2id[claim])
+                evidence_id = "C" + str(text2id[evidence])
```

Comment:
In the old version there was no Q and C I think an that's why we put `ignore_identical_ids=True`
```suggestion
                claim_id = str(text2id[claim])
                evidence_id = str(text2id[evidence])
```